rcv/frame_handlers/frc_2019.py

```diff
@@ -74,10 +73,9 @@ def prep_output_image(self):
             output = cv2.circle(output, self._points[1], 3, (0, 255, 0), -1)
         self._output_frame = output
 
-    def show_images(self):
+    def get_sliders(self):
         self.prep_output_image()
-        cv2.imshow("output", self._output_frame)
-        cv2.waitKey(1)
+        return [('original', self._frame), ('cleaned', self._cleaned_frame), ('output', self._output_frame)]
 
```

Comment:
I don't think you ever need to show all 3 images, only the output

Reply:
When i wanted to know where the error in detection these were the images I needed

Reply:
You don't need all images, you can store either the original image, or the output, and log the data into the log file with the name of the stored image. so you don't need all images